.gitignore

```diff
@@ -68,3 +68,9 @@ node_modules/
 # IDE specific
 .idea/
 *~
+
+# Country data files
+data.json
+data-*.json
+!data-fr.json
```

Comment:
maybe someone (@nburt?) can clarify to me why `data-fr.json` is in the repo anyway? No point copying it over through a refactor if there isn't a compelling reason for it to be there...

Reply:
Not sure... removing from .gitignore